src/Core/Workspace/Workspace.cs

```diff
@@ -366,6 +366,34 @@ private void LoadReferencedPackages()
             }
         }
 
+        /// <inheritdoc/>
+        public void AddProject(string projectFile)
```

Comment:
Transitive project references are calculated by `ResolveProjectReferences()`, which is called as part of `Workspace.Reload()`.

But this is a good point, since `%project foo.csproj` outputs the list of project references, and the user will (likely) expect to see all of the transitive project references listed. So we should really call `ResolveProjectReferences()` here too to ensure that the list is complete.

Reply:
Got it, the call to `ResolveProjectReferences` was precisely the piece I was missing, sorry. Thanks for the explanation!